/home/zkatz/scalable-computing-examples/04-pleasing/runinfo/000/submit_scripts/parsl.HighThroughputExecutor.block-0.1744066140.0573134.sh: line 1: workon: command not found
/usr/lib/python3.10/multiprocessing/resource_tracker.py:224: UserWarning: resource_tracker: There appear to be 7 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
